### PR TITLE
Fix inconsistent return statements

### PR DESCRIPTION
**Pull request by llFreetimell**
This commit fixes inconsistent `return` statements.

ONE-vscode-DCO-1.0-Signed-off-by: Seok NamKoong <seok9311@naver.com>

### CODE REVIEW

File: src/Execute/executeQuickInput.ts [hunk 1]
```diff
@@ -58,11 +58,11 @@ export async function runInferenceQuickInput(context: vscode.ExtensionContext) {
   async function selectInputModel(input: MultiStepInput, state: Partial<State>) {
     if (state.backend === undefined) {
       state.error = 'Backend to infer is not chosen. Please check once again.';
-      return;
+      return undefined;
```
Comment (YongseopKim) [inline src/Execute/executeQuickInput.ts:61]: Aha, I got it.

This function returns sth on below code
```ts
    return (input: MultiStepInput) => selectInputSpec(input, state);
```

So returning `undefined` explicitly is important.

**Review by YongseopKim — APPROVED** [review]
LGTM

**Review by yunjayh — APPROVED** [review]
LGTM

**Review by jyoungyun — APPROVED** [review]
LGTM

**Pull request merged**